cypress/integration/RichTextEditor.spec.ts

```diff
@@ -1139,6 +1206,73 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
 
     for (const [triggerMethod, triggerEmbeddedAsset] of methods) {
       describe(triggerMethod, () => {
+        it('adds paragraph before the block when pressing enter if the block is first child', () => {
```

Comment:
"is first child" or "is "first document node" ? Should be the latter.

There's a slight difference, a block can actually be the first child e.g. within a list item containing multiple blocks so let's be more accurate.